### PR TITLE
Add suricata.quic type

### PR DESCRIPTION
**Pull request by dispanser**
This adds schema information and some test data for `suricata.quic`, which is part of the next suricata version 7.

The associated pieces of the schema in suricata itself, this may be useful to verify against the VAST schema definition in case I missed something - we have no actual samples that contain the `cyu` and `extensions` fields, so there's no test covering those.

```json
        "quic": {
            "type": "object",
            "optional": true,
            "properties": {
                "cyu": {
                    "type": "array",
                    "items": {
                        "type": "object",
                        "properties": {
                            "hash": {
                                "type": "string"
                            },
                            "string": {
                                "type": "string"
                            }
                        },
                        "additionalProperties": false
                    }
                },
                "extensions": {
                    "type": "array",
                    "items": {
                        "type": "object",
                        "properties": {
                            "name": {
                                "type": "string"
                            },
                            "type": {
                                "type": "integer"
                            },
                            "values": {
                                "type": "array",
                                "items": {
                                    "type": "string"
                                }
                            }
                        },
                        "additionalProperties": false
                    }
                },
                "ja3": {
                    "type": "object",
                    "optional": true,
                    "properties": {
                        "hash": {
                            "type": "string"
                        },
                        "string": {
                            "type": "string"
                        }
                    },
                    "additionalProperties": false
                },
                "ja3s": {
                    "type": "object",
                    "optional": true,
                    "properties": {
                        "hash": {
                            "type": "string"
                        },
                        "string": {
                            "type": "string"
                        }
                    },
                    "additionalProperties": false
                },
                "sni": {
                    "type": "string"
                },
                "ua": {
                    "type": "string"
                },
                "version": {
                    "type": "string"
                }
            },
            "additionalProperties": false
        },
```

### CODE REVIEW

File: schema/types/suricata.schema [hunk 1]
```diff
@@ -30,6 +30,29 @@ type suricata.component.app_proto = record {
   app_proto: string
 }
 
+type suricata.component.ja3 = record {
+      hash: string #index=hash,
+      string: string,
```
Comment (tobim) [inline schema/types/suricata.schema:35]: Please align the indentation.
```suggestion
  hash: string #index=hash,
  string: string,
```

**Review by tobim — APPROVED** [review]
I would prefer to merge this today instead of waiting for a sample that contains all fields.

File: schema/types/suricata.schema (hunk 1 above)
Comment (dispanser) [reply schema/types/suricata.schema]: thanks, that slipped by :)

**Comment by tobim** [comment]
The `import-time` integration test seems to consistently fail on macOS. Any ideas?

**Pull request merged**